beginner_source/hyperparameter_tuning_tutorial.py

```diff
@@ -224,12 +224,12 @@ def train_cifar(config, checkpoint_dir=None, data_dir=None):
         train_subset,
         batch_size=int(config["batch_size"]),
         shuffle=True,
-        num_workers=8)
+        num_workers=0)
```

Comment:
Can you please explain why those are necessary?

Reply:
Just to work around the error happened when spawning threads on windows. This would not impact the performance much and we will investigate it later.

Reply:
But this changes the behavior specific to this tutorial. If this tutorial can not be validated on Windows yet, please exclude it from the testing and then work in a separate PR to fix the issue on Windows